### ISSUE TITLE
[Feature Request] Custom start and end dates

### ISSUE THREAD

**Issue by redstreet** [dangermccann/logseq-metrics#15]
Optional custom start and end dates would be great.

Useful because the typical usage with daily habits is to focus on the last month or three months or current year, or some arbitrary point in the past, as opposed to displaying all available history.

Would look like:

```
{{renderer :metrics, :exercise, Exercise, properties-line, 2023-01-01, 2023-02-28}}
```

Thank you for considering.

**Comment by dangermccann** [comment]
Good suggestion.  I'll try to work on this soon.

**Comment by dangermccann** [comment]
Implemented in 0.16.  Enjoy!

**Comment by redstreet** [comment]
Nice! I'm using it, and it works great! Thank you for doing this so quickly!
